lib/ansible/module_utils/powershell.ps1

```diff
@@ -205,6 +205,10 @@ Function Get-AnsibleParam($obj, $name, $default = $null, $resultobj = @{}, $fail
         if ($type -eq "path") {
             # Expand environment variables on path-type
             $value = Expand-Environment($value)
+            # Test if a valid path is provided
+            if (-not (Test-Path -IsValid $value)) {
```

Comment:
Not sure, but might you also want to include `-LiteralPath`? Depends on if we accept paths with wildcards as type `path` (can't remember if we discussed either way)...